### DISCUSSION TITLE
How to download files from existing server?

### DISCUSSION THREAD

**Discussion by weeebdev** [rejetto/hfs#693] [Q&A] [answered]
I want to download several folders from https://mokuro.moe/manga/
It is stated on their website, that they utilize HFS to serve their files. But I do not see any download button there.
Sorry for disturbance, I didn't find any others resources to ask this.

**Comment by PH5HP** [comment, marked as answer]
Hi,

I suppose clicking on the file will open it or download it in your browser. Somehow I doubt it that they use HFS. It seems to me a regular index of an HTML server. No signs of HFS at all. No themes which they use. Strange.

Kind regards,
Henk - https://fileserver.ph5hp.nl (if my home computer is online)

**Reply by weeebdev** [reply]
Thanks!
